### DISCUSSION TITLE
「リアクティビティー パート１」の質問

### DISCUSSION THREAD

**Discussion by tokuda109** [vuejs-jp/learn.nuxt.com#71] [ハンズオン Q&A] [answered]
https://github.com/vuejs-jp/learn.nuxt.com/blob/main/content/1.vue/2.reactivity/index.md で何か困ったことや質問があれば遠慮なく質問して下さい

**Comment by kyagi-0826** [comment]
以前一度勉強した時にreactiveだった記憶があるのですが今はrefのみになったのでしょうか？

**Reply by atlansien** [reply, marked as answer]
@kyagi-0826 
ご質問ありがとうございます！
今現在もreactiveはございます。

https://ja.vuejs.org/api/reactivity-core.html#reactive

どちらを使えばいいの？というところに関してはこちらをご参考ください
https://ja.vuejs.org/guide/extras/reactivity-transform#refs-vs-reactive-variables

**Reply by ubugeeei** [reply]
こちら，Vue Fes Japan 2023 (昨年) 行われた企画で Evan (Vue.js の作者) から直接回答もらった内容を含めて統括的にまとめられた記事です！
ご参考までに！
https://zenn.dev/comm_vue_nuxt/articles/ref-vs-reactive
